cmake/Modules/FindARM.cmake

```diff
@@ -22,6 +22,15 @@ IF(CMAKE_SYSTEM_NAME MATCHES "Linux")
       set(ASIMD_FOUND false CACHE BOOL "ASIMD/NEON available on host")
    ENDIF (ASIMD_TRUE)
 
+   #sve instruction can be found on the majority part of modern ARM processor
```

Comment:
`# SVE instruction set can be found on the majority of modern ARM processors`

Reply:
Removed. This was adapted from the similar code above however, I have removed it since it isn't really accurate here. Also I have changed SVE detection from a regex match of `/proc/cpuinfo` to an actual test compilation.